chalicelib/checks/wfr_checks.py

```diff
@@ -449,12 +725,18 @@ def checkstatus_metawfrs(connection, **kwargs):
     action.status = 'DONE'
     return action
 
-
+############################################################
+# spot_failed_metawfrs
+############################################################
```

Comment:
if one does do spot detection we don't necessarily want to get rid of the other one then - there might be times when a wfr failed for non-spot reasons but still needs to be reset. We just shouldn't queue the action automatically.

Reply:
The checks are indeed the same, but the actions are different. 
`reset_spot_failed_metawfrs` resets only those jobs that obviously failed due to spot interruptions. It uses Magma's `reset_metawfr.reset_shards`. The action is currently queued automatically and the check is run every 15 min.
`reset_failed_metawfrs` just resets everything that failed (which we might not want to do automatically, as Sarah mentioned). It uses Magma's `reset_metawfr.reset_failed`. The action is currently not queued automatically and the check is a manual check.

It's not great to have this duplicated check, but we can probably leave it as is at the moment since the functionality of `failed_metawfrs` might be useful.